### PR TITLE
Send monthly reminder about stale Transifex issues

### PR DESCRIPTION
**Pull request by Noordfrees**
It's easy for add-on designers to overlook or forget an issue, so in addition to daily updates about new issues we should also send a summary of all existing ones monthly.

### CODE REVIEW

File: wl/server/TransifexIntegration.java [hunk 1]
```diff
@@ -207,32 +219,53 @@ public synchronized void checkIssues() throws Exception {
...
 
-			Map<String, List<Issue>> relevantIssues = perUploader.get(uploader);
-			long total = 0;
-			for (List l : relevantIssues.values()) total += l.size();
+			Map<String, List[]> relevantIssues = perUploader.get(uploader);
+			long totalNew = 0;
+			long totalOld = 0;
+			for (List[] l : relevantIssues.values()) {
+				totalNew += l[0].size();
+				totalNew += l[1].size();
```
Comment (hessenfarmer) [inline wl/server/TransifexIntegration.java:227]: shouldn't one of these be totalOld ?

Comment (Noordfrees) [reply wl/server/TransifexIntegration.java]: Yes it should, fixed it

**Review by hessenfarmer — APPROVED** [review]
code looks reasonable.
not tested

**Comment by Noordfrees** [comment]
Thanks for the review :)

Merged and deployed

**Pull request merged**